chain-signatures/contract/src/lib.rs

```diff
@@ -712,6 +611,80 @@ impl VersionedMpcContract {
         }
     }
 
+    // contract version
+    pub fn version(&self) -> String {
+        env!("CARGO_PKG_VERSION").to_string()
+    }
+
+    pub fn sign_helper(&mut self, request: SignatureRequest) {
```

Comment:
@volovyks should this be marked as `[#private]`?

otherwise, people could just call it jumping over `sign` (except if there is something about using the `DATA_ID_REGISTER` that I am not aware of)